### PR TITLE
NSQ 0.3.6

### PR DESCRIPTION
**Pull request by mreiferson**
n/t

### CODE REVIEW

File: Library/Formula/nsq.rb [hunk 1]
```diff
@@ -61,16 +61,23 @@ class Nsq < Formula
...
     # build a proper GOPATH tree for local dependencies
-    (buildpath + "src/github.com/bitly/nsq").install "internal", "nsqlookupd", "nsqd", "nsqadmin"
+    (buildpath + "src/github.com/nsqio/nsq").install "Makefile", "apps", "internal", "nsqlookupd", "nsqd", "nsqadmin"
 
     ENV["GOPATH"] = buildpath
     Language::Go.stage_deps resources, buildpath/"src"
 
-    system "make"
-    system "make", "DESTDIR=#{prefix}", "PREFIX=", "install"
+    Dir.chdir(buildpath + "src/github.com/nsqio/nsq") do
```
Comment (mreiferson) [inline Library/Formula/nsq.rb]: Go 1.5 changed the way `internal` packages are evaluated, which meant that more dirs needed to be under `GOPATH`, which meant that we needed to build from within `GOPATH`.

Open to alternatives if this isn't gonna fly...

Comment (dunn) [inline Library/Formula/nsq.rb]: Should be fine, but this line can be simplified to:
```ruby
cd buildpath/"src/github.com/nsqio/nsq" do
```

**Comment by mreiferson** [comment]
updated, RFR

**Comment by DomT4** [comment]
One failure left:

```
nsq:
 * 64: Trailing whitespace was found
```

**Comment by mreiferson** [comment]
@DomT4 fixe

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/b29c2ac1dcd324eae886bb333e68ede3c9b5db3a. Thank you for your contribution to Homebrew @mreiferson; we appreciate it! :smiley_cat:

**Pull request closed without merging**